core/meson.build

```diff
@@ -1,4 +1,4 @@
-core_sources = ['src/math/matrix.cpp', 'src/math/vector.cpp', 'src/math/activation_functions.cpp', 'src/model.cpp', 'capi/cjml.cpp']
+core_sources = ['src/math/matrix.cpp', 'src/math/vector.cpp', 'src/math/activation_functions.cpp', 'src/logger.cpp', 'capi/cjml.cpp']
```

Comment:
Huh???
```suggestion
core_sources = ['src/math/matrix.cpp', 'src/math/vector.cpp', 'src/math/activation_functions.cpp', 'src/model.cpp', 'src/logger.cpp', 'capi/cjml.cpp']
```

Reply:
This was outdated

Reply:
I think merge conflicts messed this up? But I have zero clue